packages/flame/README.md

```diff
@@ -1 +1 @@
-../../README.md
\ No newline at end of file
+../../README.md
```

Comment:
What changes here?

Reply:
Probably just revert this change, I'm guessing it is some change of the symlink?
The MD linter crashes on it now:
```
Error: ENOENT: no such file or directory, open 'packages/flame/README.md'
    at Object.openSync (node:fs:596:3)
    at Object.readFileSync (node:fs:464:35)
    at lintFile (/opt/hostedtoolcache/node/18.20.1/x64/lib/node_modules/markdownlint-cli/node_modules/markdownlint/lib/markdownlint.js:852:33)
    at lintWorker (/opt/hostedtoolcache/node/18.20.1/x64/lib/node_modules/markdownlint-cli/node_modules/markdownlint/lib/markdownlint.js:937:7)
    at lintInput (/opt/hostedtoolcache/node/18.20.1/x64/lib/node_modules/markdownlint-cli/node_modules/markdownlint/lib/markdownlint.js:978:7)
    at Function.markdownlintSync [as sync] (/opt/hostedtoolcache/node/18.20.1/x64/lib/node_modules/markdownlint-cli/node_modules/markdownlint/lib/markdownlint.js:1027:3)
    at lintAndPrint (/opt/hostedtoolcache/node/18.20.1/x64/lib/node_modules/markdownlint-cli/markdownlint.js:319:35)
    at Object.<anonymous> (/opt/hostedtoolcache/node/18.20.1/x64/lib/node_modules/markdownlint-cli/markdownlint.js:325:5)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'packages/flame/README.md'
}
```

Reply:
I didn't maually edit this file. I ran `melos markdown-fix`. I believe it fixed this error: `packages\flame\README.md:1:15 MD047/single-trailing-newline Files should end with a single newline character`

Reply:
Alright, interesting that it doesn't complain about that in the pipeline, probably some platform differences. If you just checkout the `main` version it will pass in the pipeline.

Reply:
Alright, I fixed the readme file

Reply:
I'm on a Windows PC so maybe some differences...